lib/appium_lib/android/element/generic.rb

```diff
@@ -39,12 +39,16 @@ def scroll_uiselector(content, index = 0)
     # @return [Element] the element scrolled to
     def scroll_to(text, scrollable_index = 0)
       text = %("#{text}")
-
-      args = scroll_uiselector("new UiSelector().textContains(#{text})", scrollable_index) +
-             scroll_uiselector("new UiSelector().descriptionContains(#{text})", scrollable_index) +
-             scroll_uiselector(resource_id(text, "new UiSelector().resourceId(#{text});"), scrollable_index)
-
-      find_element :uiautomator, args
+      rid  = resource_id(text, "new UiSelector().resourceId(#{text});")
```

Comment:
How about the following if you'd like to ignore ` resource_id(text, "new UiSelector().resourceId(#{text});")` is `""` case?

```ruby
r_id = resource_id(text, "new UiSelector().resourceId(#{text});")
args = scroll_uiselector("new UiSelector().textContains(#{text})", scrollable_index) +
        scroll_uiselector("new UiSelector().descriptionContains(#{text})", scrollable_index)
args += scroll_uiselector(r_id, scrollable_index) unless r_id.empty?

find_element :uiautomator, args
```